internal/keycloakb/configdbmodule.go

```diff
@@ -67,3 +76,76 @@ func (c *configurationDBModule) GetConfiguration(context context.Context, realmI
 		return config, err
 	}
 }
+
+func (c *configurationDBModule) GetAuthorizations(ctx context.Context, realmID string, groupName string) ([]dto.Authorization, error) {
```

Comment:
May be reused in paper-card-service

Reply:
When we will retrieve the authorizations to apply authorizations, we will retrieve all of them not just one for a specific realm and group.